packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -120,6 +120,8 @@ class PurchaseWrapper {
   /// The payload specified by the developer when the purchase was acknowledged or consumed.
   ///
   /// The value is `null` if it wasn't specified when the purchase was acknowledged or consumed.
+  /// The `developerPayload` is removed from [BillingClientWrapper.acknowledgePurchase], [BillingClientWrapper.consumeAsync], [InAppPurchaseConnection.completePurchase], [InAppPurchaseConnection.consumePurchase]
+  /// after plugin version `0.5.0`. As a result, this will be `null` for new purchases happened after `0.5.0`.
```

Comment:
s/happened after/that happen after updating to/

Reply:
done